src/support/azext_support/custom.py

```diff
@@ -150,7 +145,7 @@ def content(self):
             body["properties"]["serviceId"] = (
                 "/providers/Microsoft.Support/services/{0}".format(service_name)
             )
-            if args.start_time == AAZUndefined:
+            if has_value(args.start_time):
```

Comment:
You have inverted the logic in your PR. The initial logic was if filter and pagination is undefined then check and now you have changed it to if filter and pagination has value then check. That means it will basically do the opposite of what it is supposed to do now

Reply:
resolved in both places